packages/mgt-components/src/components/mgt-person-card/strings.ts

```diff
@@ -8,5 +8,6 @@
 export const strings = {
   sendEmailLinkSubtitle: 'Send email',
   startChatLinkSubtitle: 'Start chat',
+  showExpandedDetailsButton: 'Show expanded details',
```

Comment:
Could you update the docs to add this string here: https://docs.microsoft.com/en-us/graph/toolkit/customize-components/localization

Reply:
Done